### PR TITLE
add(github): enable actions workflow

### PR DESCRIPTION
**Pull request by k-dovgan**

### CODE REVIEW

File: .github/workflows/precommit-check.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Universum check
+on: push
```
Comment (dogbert911) [inline .github/workflows/precommit-check.yml:2]: You use trigger on push, it works for any push in any branches.
Probably better to use next triggers - PR to master, and push to master:
```
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```

Comment (k-dovgan) [reply .github/workflows/precommit-check.yml:2]: Actually, I chose 'on: push' on purpose. This is the behavior we had before Actions: any commit to repository was checked.
And, as any commit is checked, no further checks for pull requests are required.
The only thing I don't like is that if I push more than one commit at a time, only the last one will be checked. I'd like every single commit to be checked.

Comment (axxie) [reply .github/workflows/precommit-check.yml:2]: IMO it is nice feature for users. If they want every commit checked they can push them separately. If they are only interested in the latest commit status, as they should, they can push them together.

Comment (i-keliukh) [reply .github/workflows/precommit-check.yml:2]: IMO it is nice feature for users. If they want every commit checked they can push them separately. If they are only interested in the latest commit status, as they should, they can push them together.

File: .github/workflows/precommit-check.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Setup python 3.8
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.8
+
+    - name: Install dependency
+      run: pip install universum[test]
```
Comment (i-keliukh) [inline .github/workflows/precommit-check.yml:16]: Are you sure it uses the version from the repository?

**Review by i-keliukh — APPROVED** [review]

**Pull request merged**